apps/metadata-editor-e2e/src/support/commands.ts

```diff
@@ -27,18 +27,14 @@ declare namespace Cypress {
 Cypress.Commands.add(
   'loginGN',
   (username: string, password: string, redirect = true) => {
-    cy.visit('http://localhost:8080/geonetwork/srv/eng/catalog.search#/home')
-    cy.get('.cookie-warning-actions').then(($cookie) => {
-      if ($cookie.is(':visible')) {
-        $cookie.find('button').eq(0).click()
-        cy.scrollTo('top')
-      }
+    Cypress.on('uncaught:exception', (err) => {
```

Comment:
I was hoping we could have a central place to put Cypress custom commands, instead of having to duplicate these in the different apps; or maybe from here you could import the commands from the datahub-e2e app?

Reply:
I didn't find a simple and clean way for the moment so I didn't implement it by now.

Reply:
this can be done later, no worries